### DISCUSSION TITLE
Bug Report: tzdata_release_updater ArgumentError in Plausible Container Logs

### DISCUSSION THREAD

**Discussion by onluha** [plausible/analytics#4577] [Self-Hosted Support]
Description: When running the Plausible container, an error related to the tzdata_release_updater occurs, causing the process to terminate. The issue stems from the tzdata library (version 1.1.1), where an argument passed is not in the expected tuple format, leading to an ArgumentError.

Logs:

[error] GenServer :tzdata_release_updater terminating
** (ArgumentError) errors were found at the given arguments:
  * 2nd argument: not a tuple
    :erlang.element(1, :error)
    (tzdata 1.1.1) lib/tzdata/util.ex:223: Tzdata.Util.to_int/1
    (tzdata 1.1.1) lib/tzdata/parser.ex:38: Tzdata.Parser.process_rule/1
    ...

Steps to reproduce: I used this docker compose file: https://raw.githubusercontent.com/coollabsio/coolify/main/templates/compose/plausible.yaml

**Comment by ruslandoga** [comment]
👋 @onluha 

Thank you for the report. It's already been fixed :)

**Reply by ruslandoga** [reply]
https://github.com/plausible/analytics/pull/4539
https://github.com/plausible/analytics/pull/4569

**Comment by onluha** [comment]
And which plausible docker version do I have to run for that?

**Reply by ruslandoga** [reply]
The one that would be released next: https://github.com/plausible/analytics/pull/4483 :)

**Reply by onluha** [reply]
Great! When will it be published or can I deploy an older working version and which one would that be?

**Reply by onluha** [reply]
Hi @ruslandoga do you have an update here for me? Thank you :)

**Reply by ruslandoga** [reply]
Hi! The next release would be published once it is ready. There is no ETA right now.

**Reply by ruslandoga** [reply]
You can follow the PR I linked above to keep up to date with it :)

**Reply by onluha** [reply]
Thanks @ruslandoga ! Or can I run an older version?

**Reply by ruslandoga** [reply]
There is no release with the PRs linked above right now.

**Reply by ruslandoga** [reply]
But you can run any version, it works fine. Just ignore these errors for now.

**Reply by onluha** [reply]
For me it seems to not log any events and thats the only error I see in the containers. I still see "waiting for your first pageview"

**Reply by ruslandoga** [reply]
The problem is likely elsewhere. This error is unrelated.

**Reply by mo-tech55** [reply]
Yeah. I found the issue now. It was because I ran the plausible instance without ssl and then browsers block the event logging

**Comment by ruslandoga** [comment]
> causing the process to terminate

I don't think it terminates the Plausible process. It only terminates the updater, which should be safe to ignore.
